dateparser_scripts/order_languages.py

```diff
@@ -9,7 +9,19 @@
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 
 # Languages with insufficient translation data are excluded
-avoid_languages = ['cu', 'kkj', 'nds', 'prg', 'tk', 'vai', 'vai-Latn', 'vai-Vaii', 'vo']
+avoid_languages = {
+    'ccp', 'ceb', 'cu', 'ff-Latn', 'ia', 'jv', 'kkj', 'ku', 'mi', 'nds', 'prg',
+    'sd', 'tg', 'tk', 'tt', 'vai', 'vai-Latn', 'vai-Vaii', 'vo', 'wo', 'xh',
+    'yue-Hans', 'yue-Hant'
+}
```

Comment:
You wrote here some languages from the next CLDR versions. Could you revert this?

```suggestion
avoid_languages = {'cu', 'kkj', 'nds', 'prg', 'tk', 'vai', 'vai-Latn', 'vai-Vaii', 'vo'}
```